android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStoragePackage.java

```diff
@@ -1,19 +1,21 @@
 /**
  * Copyright (c) Facebook, Inc. and its affiliates.
- *
+ * <p>
```

Comment:
Intentional?

Reply:
I still see the `<p>`?
```suggestion
 *
```

Reply:
just my linter being smart 🗡️